readthedocs/doc_builder/backends/mkdocs.py

```diff
@@ -327,27 +327,60 @@ class MkdocsHTML(BaseMkdocs):
     build_dir = '_build/html'
 
 
-class SafeLoaderIgnoreUnknown(yaml.SafeLoader):  # pylint: disable=too-many-ancestors
+class ProxyPythonName(yaml.YAMLObject):
+    def __init__(self, value):
+        self.value = value
+
+
+class SafeLoader(yaml.SafeLoader):  # pylint: disable=too-many-ancestors
 
     """
-    YAML loader to ignore unknown tags.
+    Safe YAML loader.
 
-    Borrowed from https://stackoverflow.com/a/57121993
+    This loader parses special ``!!python/name:`` tags without actually
+    importing or executing code. Every other special tag is ignored.
+
+    Borrowed from https://stackoverflow.com/a/57121993.
+    Issue https://github.com/readthedocs/readthedocs.org/issues/7461.
```

Comment:
```suggestion
    Borrowed from https://stackoverflow.com/a/57121993
    Issue https://github.com/readthedocs/readthedocs.org/issues/7461
```

Remove dots to make the link clickable.